smslibrary/src/test/java/com/eis/smslibrary/SMSDeliveredBroadcastReceiverTest.java

```diff
@@ -0,0 +1,65 @@
+package com.eis.smslibrary;
+
+import android.app.Activity;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+
+import com.eis.smslibrary.listeners.SMSDeliveredListener;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class SMSDeliveredBroadcastReceiverTest {
+    private SMSDeliveredBroadcastReceiver receiver;
+    private SMSDeliveredListener listenerMock = mock(SMSDeliveredListener.class);
+    private Context contextMock = mock(Context.class);
+    private final SMSPeer VALID_PEER = new SMSPeer("+393423541601");
+    private final String VALID_TEXT = "Some random text to send";
+    private final SMSMessage VALID_MESSAGE = new SMSMessage(VALID_PEER, VALID_TEXT);
+
+    @Before
+    public void setup(){
+        receiver = spy(new SMSDeliveredBroadcastReceiver(VALID_MESSAGE, listenerMock));
+    }
+
+    @Test
+    public void constructor_doesntThrow(){
+        new SMSDeliveredBroadcastReceiver(VALID_MESSAGE, listenerMock);
```

Comment:
The "automatic code reviewer" is in right.
Here should write:
`try {`
     `new SMSDeliveredBroadcastReceiver(VALID_MESSAGE, listenerMock);`
     `//Success }`
`catch (Exception e) `{
     `Assert.fail(...); }`

Reply:
Well, if the construction returns an error, the test will automatically fail, so I wouldn't need to check that, maybe there's a better way?

Reply:
I checked online and couldn't find any smarter way to do it, so I'll use this so that codacy won't complain